infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -129,14 +168,18 @@ public synchronized void handleMessageSetEpoch(@NonNull CorfuPayloadMsg<Long> ms
      * which monitors the exit code of Corfu. If the exit code is 100, then it resets
      * the server and DELETES ALL EXISTING DATA.
      *
-     * @param msg   The incoming message
-     * @param ctx   The channel context
-     * @param r     The server router.
+     * @param req The incoming request message.
+     * @param ctx The channel context.
+     * @param r The server router.
      */
-    @ServerHandler(type = CorfuMsgType.RESET)
```

Comment:
Now that you have replaces these message types can you also remove them from the CorfuMsgType enum.